bookwyrm/views/shelf.py

```diff
@@ -69,7 +69,8 @@ def get(self, request, username, shelf_identifier=None):
             reviews = privacy_filter(request.user, reviews)
 
         books = books.annotate(
-            rating=Subquery(reviews.values("rating")[:1])
+            rating=Subquery(reviews.values("rating")[:1]),
+            shelved_date=F("shelfbook__shelved_date"),
```

Comment:
I'm not sure if is the best way to get at the `shelved_date`, I tried accessing it via `book.shelfbook__shelved_date` in the template to no success.

Reply:
I think this is good, I don't think the templates can do that kind of advanced lookup, and it's cleaner to do it in the view anyway, imo